Makefile

```diff
@@ -1,28 +1,28 @@
-#
-#  $Id: Makefile,v 1e5e75c749c2 2015/08/08 22:03:51 fdelahoyde $
-#
-               CFLAGS:=	-O3
-            CINCLUDES:=
-              Library:=	libgswteos-10.so
-              Program:=	gsw_check
-      $(Program)_SRCS:=	gsw_check_functions.c \
-			gsw_oceanographic_toolbox.c \
-			gsw_saar.c
-      $(Library)_SRCS:=	gsw_oceanographic_toolbox.c \
-			gsw_saar.c
-      $(Library)_OBJS:=	gsw_oceanographic_toolbox.o \
-			gsw_saar.o
+Library         = libgswteos-10
+Program         = gsw_check
 
-all:	$(Program)
+PROGRAM_SOURCES = gsw_check_functions.c\
+                  gsw_oceanographic_toolbox.c\
+                  gsw_saar.c
+LIBRARY_SRCS    = gsw_oceanographic_toolbox.c \
+                  gsw_saar.c
+
+# \
```

Comment:
How does this block starting with line 10 work?  Under what conditions is it active?

Reply:
nmake and make uses different syntax for ifdef, endif etc. these directives can raise a syntax error. Since nmake does not support multiline comments for nmake case line 11,12 and so on will be active lines but for make case they will be interpreted as comment block.

By default nmake searches TOOLS.ini for definitions, by using this small trick we can include TOOLS.gcc file safely and can set all definitions for make case

Reply:
While hacky looking, it seems that the comment behavior described is documented for both [make](https://www.gnu.org/software/make/manual/html_node/Makefile-Contents.html) and [nmake](https://docs.microsoft.com/en-us/cpp/build/reference/comments-in-a-makefile).